docs/_themes/clusterhq/layout_docs.html

```diff
@@ -256,10 +261,12 @@
 
             // Is version equal to this documentation release?
             if (currentversion !== DOCUMENTATION_OPTIONS.VERSION) {
+                $('link[rel="canonical"]').attr('href', 'https://docs.clusterhq.com/en/' + DOCUMENTATION_OPTIONS.VERSION + '/' + canonicalLocation);
```

Comment:
My only concern here is if the documentation is non-latest, the canonical link is going to be defined as latest and then changed shortly after. I know you (@dwgebler) said that Google respects canonical links defined by JS however do you know how it would react to this?

I'd recommend removing the contents of line 255, and moving to an else statement as to avoid it changing during execution.